src/js/fluxcontroller.js

```diff
@@ -374,13 +384,14 @@ define(function (require, exports, module) {
      * @return {ActionReceiver}
      */
     FluxController.prototype._makeActionReceiver = function (proto, action, actionName) {
-        if (!action.writes) {
+        var actionObject = action.action;
+        if (actionObject && !actionObject.writes) {
```

Comment:
i'm not sure we're at a point where we have properly filtered out all the of the actions from the broader "actions". i think we'll need to wait to make that change